charger/delta.go

```diff
@@ -275,18 +275,6 @@ func (wb *Delta) Identify() (string, error) {
 	return bytesAsString(b), nil
 }
 
-var _ api.Battery = (*Delta)(nil)
```

Comment:
Template says "ISO". Iso allow retrieving soc over DC (which is what OpenWB Pro does). If this does not work here- does the charger even support ISO? Which functionality? Vehicle MAC?

Reply:
According to their website Delta offers 2 AC chargers (that do not support SoC). The available DC chargers range von 25-200kW and are proably not suited for home use. Thats why I removed the SoC feature from the Delta plugin with 3ddf904.

Reply:
If we wanted to keep the support for Delta DC chargers we would have to introduce some type of configuration setting or detection mechanism to distinguish AC from DC chargers because right now the SoC from the charger (that always reports 0% SoC) keeps overwriting the real SoC from vehicle.

Reply:
If none of the ISO features are used we should probably remove it from the template, too.

Reply:
Ah sorry, did not get you were refering to the template. You are right of course, removed ISO from template with ab781f4